src/index.ts

```diff
@@ -21,8 +21,15 @@ export interface PyroscopeConfig {
   configured: boolean
 }
 
-const INTERVAL = 10000
-const SAMPLERATE = 100
+// https://github.com/google/pprof-nodejs/blob/0eabf2d9a4e13456e642c41786fcb880a9119f28/ts/src/time-profiler.ts#L37-L38
+// The Interval in which samples should be collected. (in microseconds)
+// converting to microseconds
+const SAMPLE_RATE =  Number(process.env['PYROSCOPE_SAMPLE_RATE'])*10 || 10000 // in microseconds -> ex. 10000 microseconds --> 0.01 seconds --> 100Hz
+
+// The Duration for which a sample should be collected.
+// https://github.com/google/pprof-nodejs/blob/0eabf2d9a4e13456e642c41786fcb880a9119f28/ts/src/time-profiler.ts#L35-L36
+const SAMPLING_DURATION =  process.env['PYROSCOPE_SAMPLING_DURATION'] || 100 // in milliseconds
```

Comment:
This parameter in essence says profiler how long to be collecting samples to create a single profile. I might be mistaken, but it should be 10s:
```js
const SAMPLING_DURATION =  process.env['PYROSCOPE_SAMPLING_DURATION'] || 10000 // in milliseconds
```

For consistency with the existing pyroscope profiler, I'd call the option `UPLOAD_INTERVAL` (same for env var), expressed in seconds.